### PR TITLE
Fix build, add Rails 6 support, and update ruby versions

### PR DESCRIPTION
**Pull request by p8**
Removes ruby 2.3.8 from the build matrix because it is end-of-life and
fails with Rails 6.

### CODE REVIEW

**Comment by bluemihai** [comment]
Hi @p8 thanks for this...

I wanted to add Foundation to my Rails 6 app and was struggling (brand new to webpacker, not a front-end dev by any means).

Does your pull request fix it?
If yes, is there any way for me to roll my own before it gets merged and included in the gem?
(I'm also fairly new to open-source and github, so thanks for bearing with me)

Thanks!

**Review by chnikki — APPROVED** [review]

**Comment by p8** [comment]
Hi @bluemihai,

You can reference my fork by using the following in your Gemfile:
```
gem 'foundation-rails', git: 'https://github.com/p8/foundation-rails.git', ref: '199beb3' # switch to sassc and fixes reveal animation
```

File: .travis.yml [hunk 1]
```diff
@@ -3,9 +3,9 @@ sudo: false
 cache: bundler
 
 rvm:
-  - 2.3.8
-  - 2.4.5
-  - 2.5.3
+  - 2.4.9
+  - 2.5.7
+  - 2.6.5
```
Comment (swiknaba) [inline .travis.yml:8]:
```suggestion
  - 2.4.10
  - 2.5.8
  - 2.6.6
  - 2.7.2
  - 3.0.0
```

Comment (swiknaba) [reply .travis.yml:8]: this did not age well 🤣

**Pull request closed without merging**